src/app/admintoolbar/admintoolbar.component.css

```diff
@@ -0,0 +1,9 @@
+.navbar{
```

Comment:
Add space before start bracket in line 1.

Reply:
Done.